app/src/quizSubmit.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+try:
+    # Pulling json data from "questions.json"
+    with open("json/questions.json", "r") as f:
+        data = json.load(f)
+except IOError:
+    print("questions.json file not found")
+    exit()
+
+# Created for loop to iterate for all the questions defined in "questions.json"
+for i in range(data["module"]["questions"][-1]["number"]):

```

Comment:
Format is a bit confusing, since you have a num_question attribute, would it not be easier to run it for the range of num_questions?

Reply:
Yes, you're right. I ended up adding num_questions as an attribute after creating that for loop so I forgot to use it for the range